vertical-pod-autoscaler/pkg/updater/main.go

```diff
@@ -51,9 +50,6 @@ import (
 )
 
 var (
-	updaterInterval = flag.Duration("updater-interval", 1*time.Minute,
-		`How often updater should run`)
```

Comment:
This one is being removed, should it be?

Reply:
Oh, I see it was changed for `--interval`
My vote is to not change any parameters, just move the common ones

Reply:
Sure, will be fixed